src/modules/core/components/Popover/Popover.tsx

```diff
@@ -116,31 +120,42 @@ const Popover = ({
 
   const lastIsOpenProp = usePrevious(isOpenProp);
 
+  const close = useCallback(
+    (data?: any, modifiers?: { cancelled: boolean }) => {
+      if (openTimeoutRef.current) {
+        clearTimeout(openTimeoutRef.current);
+      }
+      if (closeTimeoutRef.current) {
+        clearTimeout(closeTimeoutRef.current);
+      }
+      _setIsOpen(false);
```

Comment:
I'm not really a fan of prepending the variable name with an underscore. 

Any particular reason why you did that ?

Reply:
Please do. I see no reason for it to be like this